tests/test_transforms.py

```diff
@@ -117,19 +117,31 @@ def test_tth_to_q_bad(wavelength, tth, expected_error_type, expected_error_msg):
 
 
 @pytest.mark.parametrize(
-    "q, expected_d",
+    "q, expected_d, warning_expected",
     [
-        # UC1: User specified empty q values
-        (np.array([]), np.array([])),
-        # UC2: User specified valid q values
+        # Case 1: empty q values, expect empty d values
```

Comment:
Let's put a high level contextual statement always to make it easier to review.  To see what I mean you could look at my edits to the PR on scale_to and __eq__.  

 Here it would be something like
```
# test conversion of q to d with good values
# Case 1: empty q values, expect empty d values

...

# Case 2: valid q values, expect d values without warning

...

Case 3.  valid q values containing 0, expect d values with divide by zero warning 

```

Reply:
done